src/components/panel/panel.spec.js

```diff
@@ -1139,18 +1164,38 @@ describe('$mdPanel', function() {
           expect(getGroupPanels('test')).not.toContain(panel);
         });
 
-    it('should remove a panel from a group on panel destroy', function() {
```

Comment:
Why are you removing the single group tests? The config is different, so I think you still need a test or two here.